dev/tasks/dev/utils/git.js

```diff
@@ -142,5 +143,20 @@ module.exports = {
 		];
 
 		tools.shExec( commitCommands.join( ' && ' ) );
+	},
+
+	/**
+	 * Adds remote to repository under specified path.
+	 *
+	 * @param {String} pluginName
+	 * @param {String} repositoryPath
+	 */
+	addRemote( pluginName, repositoryPath ) {
+		const addRemoteCommands = [
+			`cd ${ repositoryPath }`,
+			`git remote add ${ defaultOrigin } git@github.com:ckeditor/${ pluginName }.git`
```

Comment:
GitHub url cannot be hardcoded. See the variables available in `create-package.js` - there's `gitHubUrl`.